src/System.Data.SqlClient/tests/ManualTests/SQL/ConnectionPoolTest/PoolBlockPeriodTest.netcoreapp.cs

```diff
@@ -9,22 +9,23 @@ namespace System.Data.SqlClient.ManualTesting.Tests
 {
     public class PoolBlockPeriodTest
     {
-        private static readonly string _sampleAzureEndpoint = "nonexistance.database.windows.net";
+        private static readonly string _sampleAzureEndpoint = "nonexistent.database.windows.net";
```

Comment:
This can be `const` instead of `static readonly`, then renamed to `SampleAzureEndpoint`, and then this can be used in the InlineData attributes rather than repeating this value multiple times.